R/compare.R

```diff
@@ -80,15 +90,7 @@ compare_output <- function(model, pars_obs, data, type="sircovid_basic") {
           ll_nbinom(data$deaths[t], phi_death * model_deaths, k_death, exp_noise)
       }
     } else {
-      if (!is.na(data$deaths[t])) {
-        ## new deaths summed across ages/infectivities
-        model_deaths_hosp <- colSums(state[index_D_hosp, ]) -
-          colSums(prev_state[index_D_hosp, ])
-        model_deaths_comm <- colSums(state[index_D_comm, ]) -
-          colSums(prev_state[index_D_comm, ])
-        log_weights <- log_weights +
-          ll_nbinom(data$deaths[t], phi_death_hosp * model_deaths_hosp + phi_death_comm * model_deaths_comm, k_death, exp_noise)
-      } else {
+      if (!is.na(data$deaths_hosp[t] || !is.na(data$deaths_comm[t]))){
```

Comment:
I think the bracket is in the wrong place here? i.e. should be  

```suggestion
      if (!is.na(data$deaths_hosp[t]) || !is.na(data$deaths_comm[t])){
```